src/main/java/teammates/ui/template/StudentCommentsFeedbackResponseCommentRow.java

```diff
@@ -0,0 +1,17 @@
+package teammates.ui.template;
+
+public class StudentCommentsFeedbackResponseCommentRow extends FeedbackResponseCommentRow {
```

Comment:
Yup, for now I just extended the FeedbackResponseCommentRow class because I reused the `ResponseRow` data structure, which contains a list of `FeedbackResponseCommentRow`.
Otherwise, I would have to create a new `*ResponseRow` data structure to contain a list of `StudentCommentsFeedbackResponseCommentRow`, and so on for the outer loops. I understand its not very consistent now, but it will be worked on in issue #3839 when this is done :smiley: